test/macros/spawn_lnd_daemon.js

```diff
@@ -0,0 +1,235 @@
+const removeDir = require('rimraf');
+const {spawn} = require('child_process');
+const uuidv4 = require('uuid/v4');
+const asyncAuto = require('async/auto');
+const rpcServerReady = /password RPC server listening on|Finished rescan for 0 addresses/;
+const namespaceFail = /Shutdown complete|Done generating TLS certificates/;
+
+const exec = require('child_process').exec;
+const path = require('path');
+const fs = require('fs');
+const {createAddress} = require('ln-service');
```

Comment:
Style is to separate node module requires from built-in packages - and to use destructuring, and to put built in things and modules up above regular vars